rest-api/src/main/resources/paths/studies/v5_studies_studyId_participants_self_timeline.yml

```diff
@@ -22,6 +22,10 @@ get:
             $ref: ../../responses/304.yml
         401:
             $ref: ../../responses/401.yml
+        404:
+            description: The study does not have a defined schedule.
+            schema:
+                $ref: ../../definitions/message.yml
```

Comment:
Noting optional reference change.

Reply:
I've updated these—I forgot to check in the files